src/components/ResolutionToolbar.vue

```diff
@@ -1,34 +1,58 @@
 <template>
     <div>
-        <v-btn id="grid-button" class="pink white--text" fab @click="showProperties()" @mouseenter.native="hover = true" @mouseleave.native="hover = false">
-            <span class="material-icons">grid_on</span>
-        </v-btn>
-        <v-btn v-if="hover" id="grid-hover" class="grey white--text" x-small depressed>Grid Settings</v-btn>
+        <v-tooltip left>
+            <template v-slot:activator="{ on, attrs }">
+                <v-btn
+                    id="grid-button"
+                    v-bind="attrs"
+                    class="pink white--text"
+                    fab
+                    v-on="on"
+                    @click="showProperties()"
+                    @mouseenter.native="hover = true"
+                    @mouseleave.native="hover = false"
+                >
+                    <span class="material-icons">grid_on</span>
+                </v-btn>
+            </template>
+            <span>Grid Settings</span>
+        </v-tooltip>
         <div v-if="activated" id="resolution-toolbar">
             <v-switch v-model="slider_enabled" color="#304FFE" hide-details @change="clickedGrid">
                 <template v-slot:label class="mdl-switch__label">Enable Automatic Grid</template>
             </v-switch>
             <v-switch v-model="switch2" color="#304FFE" @change="clickedSnap">
                 <template v-slot:label class="mdl-switch__label">Render Snap Points</template>
             </v-switch>
-            <veeno ref="slider" :disabled="slider_enabled" v-bind="sliderOptions" @change="updateGrid" />
+            <veeno ref="slider" v-model="sliderValue" :disabled="slider_enabled" v-bind="sliderOptions" @change="updateGrid" />
         </div>
+        <div id="bottom-info-bar">Grid Size: {{ sliderValue }} &mu;m</div>
     </div>
 </template>
 
 <script>
+// Issues: Beining Aug.5.21
+// Grid Settings slider out of range
+// "console log : Could not set the Zoom
+
 import veeno from "veeno";
 import "nouislider/distribute/nouislider.min.css";
 import Registry from "../app/core/registry";
 import wNumb from "wnumb";
+import EventBus from "@/events/events";
 
 export default {
     name: "ResolutionToolbar",
     components: {
         veeno
     },
 
-    props: {},
+    props: {
+        sliderValue: {
+            type: Number,
+            default: 1000
+        }
+    },
```

Comment:
We should be passing this as a prop since the initial value should be loaded from the `Registry.viewManager`